proprietary/tokens/src/common/amsterdam/link-type.tokens.json

```diff
@@ -0,0 +1,47 @@
+{
+  "amsterdam": {
+    "link-type": {
```

Comment:
A bit long maybe, but `link-appearance` might be perfect.

If not, [synonyms here](https://www.thesaurus.com/browse/appearance). Presentation?